pwiz/data/vendor_readers/Thermo/Reader_Thermo_Test.cpp

```diff
@@ -199,6 +199,9 @@ int main(int argc, char* argv[])
         config.peakPicking = true;
         pwiz::util::testReader(reader, testArgs, testAcceptOnly, requireUnicodeSupport, IsRawFile(), config);
 
+        config.addNoiseData = true;
+        pwiz::util::testReader(reader, testArgs, testAcceptOnly, requireUnicodeSupport, IsRawFile(), config);
```

Comment:
Use `IsNamedRawFile()` to restrict the noise data test to 1 or 2 RAW files: we don't need tests for all the single scan RAW Files we have.